pkg/kubelet/status/status_manager.go

```diff
@@ -372,6 +372,10 @@ func (m *manager) syncBatch() {
 		for uid, status := range m.podStatuses {
 			syncedUID := uid
 			if mirrorUID, ok := podToMirror[uid]; ok {
+				if mirrorUID == "" {
+					glog.V(5).Infof("Static pod %q (%s/%s) does not have a corresponding mirror pod; skipping", uid, status.podName, status.podNamespace)
```

Comment:
nit: follow the same format used for logging elsewhere: https://github.com/kubernetes/kubernetes/blob/7c27a2686ff70a11bda20c27c9cc8dc154183357/pkg/kubelet/util/format/pod.go#L34

Better yet, add a helper function in that package that takes a the name, namespace, and UID.